packages/dendron-cli/src/commands/notes.ts

```diff
@@ -145,6 +157,23 @@ export class NoteCLICommand extends CLICommand<CommandOpts, CommandOutput> {
           this.print(`deleted ${note.fname}`);
           return { data: { payload: note.fname, rawData: resp } };
         }
+        case NoteCommands.MOVE:
+        case NoteCommands.RENAME: {
+          const { query, vault } = checkQueryAndVault(opts);
+          const note = NoteUtils.getNoteOrThrow({
+            fname: query,
```

Comment:
semantically, `query` is meant to be a lookup query which means that it doesn't have to match the path exactly. the way its being used here, we are actually looking for an exact match. 
in order to be explicit, I would use a new argument here  (eg. `srcFname`). this also means we want to remove the `checkQueryAndVault` check and create new function `eg. checkArgsForMoveCommand`